playbooks/cron.yml

```diff
@@ -0,0 +1,8 @@
+---
+- hosts: common
+  tasks:
+    - name: Add common cron jobs
+      include_role:
+        name: common
+        defaults_from: cron
+        tasks_from: cron
```

Comment:
Following the previous pattern the ES stuff would then be

```yaml
- hosts: elasticsearch
  tasks:
    # 1. Template out the ES curator script
    # 2. Add the cron job
```

Reply:
The problem I run into with this one is that ansible can't seem to find the path of the curator's jinja script. The easier fix was to create an elasticsearch tasks file and include the role. This isn't great for consistency, but it does seem to do the trick.